ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -80,7 +80,7 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
     // When staggered scheme is adopted, nodal pressure and nodal concentration
     // are accessed by process id.
     static const int hydraulic_process_id = 0;
-    static const int transport_process_id = 1;
+    static const int first_transport_process_id = 1;
```

Comment:
IMHO, for consistent naming scheme, it would be better to revert this name change.

Reply:
`first_transport_process_id` is used for calculating fluid density, with the assumption that fluid density is dependent upon the concentration of the component which is placed uppermost. There is no such an issue in single component transport simulation.

Reply:
Then I recommend to add a comment explaining the variable purpose (if it cannot be explained by variable name).
```c++
    // TODO (renchao-lu): The variable is used in the calculation of the fluid's
    // density and flux indicating the transport process id. For now it is
    // assumed that these quantities depend on the first occurring transport
    // process only. The density and flux calculations have to be extended to
    // all processes.
```
*sic*!